src/search-scenario/azext_search_scenario/custom.py

```diff
@@ -0,0 +1,312 @@
+    params = [param for param in params if param and param != '']
+    for param in params:
+        store_true_params = ['-h', '--yes', '-y', '--no-wait', '--dry-run', '--no-log']
+        if param in store_true_params:
+            args.append(param)
+        else:
+            print_styled_text([(Style.ACTION, "Please input "), (Style.PRIMARY, param + ": ")], end='')
+            value = input()
+            if param == '<positional argument>':
+                if value:
+                    args.append(value)
+            else:
+                args.append(param)
+                if value:
+                    args.append(value)
+                else:
+                    args.pop()
+
+    output_format = ctx_cmd.cli_ctx.config.get('search_scenario', 'output', fallback='status')
+
+    if '--output' not in args and '-o' not in args:
+        args.append('--output')
+        if output_format == 'status':
+            is_show_operation = False
+            for operation in ['show', 'list', 'get', 'version']:
+                if operation in command:
+                    is_show_operation = True
+                    break
+            if is_show_operation:
+                args.append('json')
+            else:
+                args.append('none')
+        else:
+            args.append(output_format)
+
+    if not catch_exception:
+        exit_code = ctx_cmd.cli_ctx.invoke(args)
+
+    else:
+        try:
+            exit_code = ctx_cmd.cli_ctx.invoke(args)
+        except Exception:   # pylint: disable=broad-except
+            return -1
+        except SystemExit:
+            return -1
+
+    if output_format == 'status' and exit_code == 0:
+        print()
+        from .utils import print_successful_styled_text
+        print_successful_styled_text('command completed\n')
+
+    return exit_code
+
+
+def _get_command_item_sample(command):
+    if "example" in command and command["example"]:
+        command_sample, _ = _parse_argument_value_sample(command["example"].replace(" $", " "))
+        return command_sample
+
+    from knack import help_files
+    parameter = []
+    if "arguments" in command and command["arguments"]:
+        parameter = command["arguments"]
+        sorted_param = sorted(parameter)
+        cmd_help = help_files._load_help_file(command['command'])   # pylint: disable=protected-access
+        if cmd_help and 'examples' in cmd_help and cmd_help['examples']:
+            for cmd_example in cmd_help['examples']:
+                command_sample, example_arguments = _parse_argument_value_sample(cmd_example['text'])
+                if sorted(example_arguments) == sorted_param:
+                    return command_sample
+
+    command = command["command"] if command["command"].startswith("az ") else "az " + command["command"]
+    command_sample = f"{command} {' '.join(parameter) if parameter else ''}"
+    return [(Style.PRIMARY, command_sample)]
+
+
+def _parse_argument_value_sample(command_sample):
```

Comment:
What does this function do?
format sample? or parse argument?
Let function name straightforward.

Reply:
Done. Rename the function with `_format_sample`.